ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-isis.cfg

```diff
@@ -108,6 +108,12 @@ router isis EVPN_UNDERLAY
    set-overload-bit on-startup wait-for-bgp
    advertise passive-only
    spf-interval 250
+   timers csnp generation interval 10 seconds
```

Comment:
can you improve coverage for your changes?

Reply:
Fixed